### DISCUSSION TITLE
不好意思，请问是否有办法修改这些样式？

### DISCUSSION THREAD

**Discussion by Mantyke** [wangchucheng/hugo-eureka#133] [Q&A] [answered]
不好意思，不太熟悉相关操作，想问一下，是否有方法修改主题在白昼/夜晚状态下的底色？白色显得太亮，而黑色显得太黑，想修改成比较柔和的配色。
另外，在长段文章中，正文的行高也显得过矮，造成文章看起来非常密集。
在其他的模板中，可以修改相关的CSS文件，但是我在Eureka里没能找到这些参数

**Comment by wangchucheng** [comment, marked as answer]
目前暂时无法修改颜色的具体设置，之后有计划增加自定义色彩模式。但由于时间问题，可能需要一段时间。同时也很欢迎所有人提出相关的PR！

**Reply by Mantyke** [reply]
谢谢回复，期待更新~
